### DISCUSSION TITLE
Release schedule docker image

### DISCUSSION THREAD

**Discussion by markkvdb** [open-meteo/open-meteo#641] [Q&A] [answered]
How and when are the docker images updated? I saw that a bug fix was pushed today but no subsequent update was released for the docker images.

Would it make sense to automatically create a new image on every commit (with the commit id as the version) and link the release versions to the image versions?

**Comment by markkvdb** [comment, marked as answer]
I looked in the github actions and saw that the release is triggered by git tags. So, I guess it depends on when a new git tag is pushed!

**Reply by patrick-zippenfenig** [reply]
Correct. I am also planing to integrate semantic release to automate it further, but I haven't found a good solution to build DPKG and Docker packages with it in one go.
